pom.xml

```diff
@@ -18,10 +18,10 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus-plugin.version>1.8.1.Final</quarkus-plugin.version>
+        <quarkus-plugin.version>1.12.2.Final</quarkus-plugin.version>
```

Comment:
This version should be aligned with the PR128 ;-)

Reply:
Yes, but since PR #128 requires additional changes, I stayed in this version which is the one required to create the test profiles. We can always cherry those changes if it is approved before this PR.